server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/bittrex/BittrexExchange.java

```diff
@@ -333,7 +334,9 @@ public BigDecimal calculateBuyPrice(String cryptoCurrency, String fiatCurrency,
         try {
             MarketDataService marketDataService = getExchange().getMarketDataService();
             RateLimiter.waitForPossibleCall(getClass());
-
+            if (CryptoCurrency.BTBS.getCode().equals(cryptoCurrency)) {
+                fiatCurrency = CryptoCurrency.USDT.getCode();
+            }
```

Comment:
can the fiatCurrency parameter be anything other than USD? this exchange only supports USD and it's not possible to configure it using a parameter from admin, right? Just making sure

Reply:
Yes, it should be possible to select cryptocurrency from admin even when bittrex support just exchange between USDT and BTBS. I need to upgrade it.